src/docgen/render/json.ts

```diff
@@ -4,7 +4,7 @@
 export class Json<T> {
   constructor(public readonly content: T) {};
 
-  public render(replacer?: any, space?: string | number): string {
-    return JSON.stringify(this.content, replacer, space);
+  public render(): string {
+    return JSON.stringify(this.content, null, 2);
```

Comment:
I included these options in the original API, but in retrospect it's a bit clunky and I think it makes more sense to render the document with indentation by default. If someone wants the old behavior they can just access the public `content` field directly